### PR TITLE
Filter out a black list of files from manifest bundle

### PR DESCRIPTION
**Pull request by galletti94**
/cc @kevinrizza @shawn-hurley

### CODE REVIEW

File: operatorcourier/push.py [hunk 1]
```diff
@@ -8,9 +8,10 @@
     OpCourierQuayCommunicationError,
     OpCourierQuayErrorResponse
 )
+from operatorcourier.manifest_parser import filterOutFiles
 
 logger = logging.getLogger(__name__)
-
+BLACK_LIST = ["art.yaml", "image-references"]
```
Comment (kevinrizza) [inline operatorcourier/push.py]: Can we comment this to describe what it is for?

File: operatorcourier/push.py [hunk 2]
```diff
@@ -28,6 +29,7 @@ def push(self, bundle_dir, namespace, repository, release, auth_token):
         :param auth_token: Authentication token used to push to Quay.io.
         """
         logger.info('Generating 64 bit bundle and pushing to app registry.')
+        filterOutFiles(bundle_dir, BLACK_LIST)
```
Comment (galletti94) [inline operatorcourier/push.py]: Note that this removes files from a temporary directory created at runtime (not the actual directory)

File: operatorcourier/manifest_parser.py [hunk 3]
```diff
@@ -105,3 +105,13 @@ def get_csvs_pkg_info_from_root(source_dir: str) -> Tuple[List[Tuple], Tuple]:
 def is_yaml_file(file_path):
     yaml_ext = ['.yaml', '.yml']
     return os.path.splitext(file_path)[1] in yaml_ext
+
+def filterOutFiles(folder_path, file_names):
```
Comment (kevinrizza) [inline operatorcourier/manifest_parser.py]: Can we document this function?

**Pull request merged**